photon-ml/src/integTest/scala/com/linkedin/photon/ml/util/PhotonLoggerIntegTest.scala

```diff
@@ -62,14 +62,14 @@ class PhotonLoggerIntegTest extends SparkTestUtils with TestTemplateWithTmpDir {
       logger.warn("test message 5")
 
     } finally {
-      logger.close
+      logger.close()
     }
 
-    val fs = FileSystems.getDefault()
+    val fs = FileSystems.getDefault
     assertTrue(Files.exists(fs.getPath(logFile)))
 
-    val lines = Source.fromFile(logFile).getLines.toArray
-    assertEquals(lines.size, 5)
+    val lines = Source.fromFile(logFile).getLines().toArray
+    assertEquals(lines.length, 5)
```

Comment:
@joshvfleming For this change, I actually agrees with my IntelliJ editor that getLines() is defined as an empty paren method (in http://www.scala-lang.org/api/rc2/scala/io/BufferedSource.html), and was accessed as parameterless.

For the preference for length over size: calling size may require additional implicit conversion to SeqLike to be made. Calling length on Array or Strings may provide advantages. To further verify this, I found a few similar questions asked on StackOverflow, for example: http://stackoverflow.com/questions/19826532/do-size-and-length-in-scala-collections-generate-the-same-byte-code. There are a few more discussions there :) 

But I agree with you that these are very minor things, and I don't have very strong preference neither.